gamestonk_terminal/portfolio/portfolio_model.py

```diff
@@ -0,0 +1,274 @@
+"""Portfolio Model"""
+__docformat__ = "numpy"
+
+import os
+import math
+from datetime import date, timedelta
+from typing import List
+
+import numpy as np
+import pandas as pd
+
+from gamestonk_terminal.portfolio import portfolio_view, yfinance_model
+
+# pylint: disable=E1136
+# pylint: disable=unsupported-assignment-operation
+
+
+def save_df(df: pd.DataFrame, name: str) -> None:
+    """
+    Saves the portfolio as a csv
+
+    Parameters
+    ----------
+    df : pd.DataFrame
+        The dataframe to be saved
+    name : str
+        The name of the string
+    """
+    path = os.path.dirname(os.path.abspath(__file__))
+    path = os.path.abspath(os.path.join(path, "portfolios", name))
+    if ".csv" in name:
+        df.to_csv(path, index=False)
+    elif ".json" in name:
+        df.to_json(path, index=False)
+    elif ".xlsx" in name:
+        df = df.to_excel(path, index=False, engine="openpyxl")
+
+
+def load_df(name: str) -> pd.DataFrame:
+    """
+    Saves the portfolio as a csv
```

Comment:
Loads portfolio.  Also docstring styling would be 

```
"""Loads portfolio from df
```

Applies to most things here